http/src/main/resources/init-resources/cloud-init.yml

```diff
@@ -0,0 +1,17 @@
+#cloud-config
+
+bootcmd:
+- fsck.ext4 -tvy /dev/sdb
+- mkdir -p /mnt/disks/work
+- mount -t ext4 -O discard,defaults /dev/sdb /mnt/disks/work
+
+write_files:
```

Comment:
I am not sure if this is really needed but maybe we should do the same thing for `startup` so the startup script is guaranteed to run after docker?
```
- path: /etc/systemd/system/google-startup-scripts.service.d/override.conf 
  permissions: 0644
  owner: root    
  content:
    [Unit]      
    After=docker.service
```

Reply:
this file exists across vm restarts..so I think we're good :)